include/rosdiscover-clang/Ast/Stmt/ApiCall.h

```diff
@@ -117,31 +122,29 @@ class RateSleep : public SymbolicRosApiCall {
 
 class Publish : public SymbolicRosApiCall {
 public:
-  Publish(std::string const &publisher, std::vector<std::unique_ptr<SymbolicControlDependency>> controlDependencies={}) : 
+  Publish(std::string const &publisher, std::unique_ptr<SymbolicExpr> controlDependencies = std::make_unique<BoolLiteral>(true)) : 
```

Comment:
What does it mean for control dependencies to be the literal "true"? Control dependencies suggests a collection of some item, whereas the SymbolicExpr suggests something else. Renaming this argument (for all of the relevant classes) might be helpful.

Reply:
fixed